test/patch/element.spec.js

```diff
@@ -113,6 +113,17 @@ describe('element', function () {
     expect(eventListener.handleEvent).not.toHaveBeenCalled();
   });
 
+  it('should have no effect while calling addEventListener / removeEventListener without listener', function () {
+    var eventListener;
+    expect(function() {
```

Comment:
I test in IE9/10/11 addEventListener('click', {a: 2})  , nothing happened, no error was thrown.
![image](https://cloud.githubusercontent.com/assets/3468483/10708528/8ee9a5ce-7a3e-11e5-92ef-adfa7bc34d65.png)